v1/angler.json

```
@@ -0,0 +1,113 @@
+      "title": "Reboot to Bootloader",
+      "description": "Press and hold the volume down and power buttons until the phone reboots.",
+      "image": "phone_power_down",
+      "button": true
+    },
+    "confirm_model": {
+      "title": "Confirm your model",
+      "description": "Please confirm that your device is a Huawei Nexus 6P (angler). The LG Nexus 6 (shamu) is not compatible!"
+    },
+    "confirm_vendorimage": {
+      "title": "Confirm Android version",
+      "description": "Please confirm that your device has been reset to stock Android 7.1.2 - Critical vendor files are otherwise missing!"
+    }
+  },
+  "operating_systems": [
+    {
+      "name": "Ubuntu Touch",
+      "sanity_check": "Are you sure?",
+      "options": [
+        {
+          "var": "channel",
+          "name": "Channel",
+          "tooltip": "The release channel",
+          "link": "https://docs.ubports.com/en/latest/about/process/release-schedule.html",
+          "type": "select",
+          "remote_values": { "type": "systemimagechannels" }
+        },
+        {
+          "var": "wipe",
+          "name": "Wipe Userdata",
+          "tooltip": "Wipe personal data",
+          "type": "checkbox"
+        },
+        {
+          "var": "bootstrap",
+          "name": "Bootstrap",
+          "tooltip": "Flash system partitions using fastboot",
+          "type": "checkbox",
+          "value": true
+        }
+      ],
+      "prerequisites": [],
+      "steps": [
+        {
+          "type": "adb:reboot",
+          "condition": {"var": "bootstrap", "value": true},
+          "to_state": "bootloader",
+          "fallback_user_action": "bootloader"
+        },
+        {
+          "type": "adb:reboot",
+          "condition": {"var": "bootstrap", "value": false},
+          "to_state": "recovery",
+          "fallback_user_action": "recovery"
+        },
+        {
+          "type": "download",
+          "condition": {"var": "bootstrap", "value": true},
+          "group": "firmware",
+          "files": [
+            {
+              "url": "https://ci.ubports.com/job/Device%20Compatibility%20Images/job/halium-angler/14/artifact/halium-unlocked-recovery_angler.img",
+              "checksum": "32d73944c4d5d2f6fb6f7f06b0a3bec1e9509fbe2c6e5965453c1982facde38d"
+            }
+          ]
+        },
+        {
+          "type": "fastboot:flash",
+          "condition": {"var": "bootstrap", "value": true},
+          "flash": [
+            {
+              "partition": "recovery",
+              "file": "halium-unlocked-recovery_angler.img",
+              "group": "firmware"
+            },
+            {
+              "partition": "boot",
+              "file": "halium-unlocked-recovery_angler.img",
+              "group": "firmware"
+            }
+          ]
+        },
+        {
+          "type": "fastboot:reboot",
+          "condition": {"var": "bootstrap", "value": true},
+          "fallback_user_action": "boot",
```

```suggestion
          "fallback_user_action": "boot"
```

if you commit the one below, so it's still valid json